packages/common/src/resources/convert-storage-resource-to-item-resource.ts

```diff
@@ -45,8 +45,6 @@ export function convertStorageResourceToItemResource(
   } else if (folder.endsWith("_info_metadata")) {
     type = EFileType.Metadata;
     filename = "metadata.xml";
-  } else if (folder.endsWith("_info")) {
```

Comment:
What happens to existing Solution Template items that have `_info` resources?

Reply:
Because those files are redundant with (and actually a subset of) the same files in the data-section zip files, they are ignored.